apps/native/app/src/screens/wallet-pass/wallet-pass.tsx

```diff
@@ -167,6 +172,21 @@ export const WalletPassScreen: NavigationFunctionComponent<{
   const barcodeHeight = barcodeWidth / 3
   const updated = data?.fetch?.updated
 
+  const { loading } = res
+
+  const informationTopSpacing =
+    (allowLicenseBarcode && ((loading && !data?.barcode) || data?.barcode)) ||
+    (!isConnected && isBarcodeEnabled)
+      ? barcodeHeight + LICENSE_CARD_ROW_GAP
+      : 0
+
+  const [key, setKey] = useState(0)
+  useEffect(() => {
+    // Used to rerender ScrollView to have correct ContentInset based on barcode/no barcode
+    // Remove once barcodes are live
+    setKey((prev) => prev + 1)
+  }, [isBarcodeEnabled])
```

Comment:
The feature flag is super fast to go from false to true initially so this should be super fast.